### PR TITLE
disable source generation for rhel7-based 9.6 version

### PR DESCRIPTION
**Pull request by pkubatrh**
Resolves: #392

### CODE REVIEW

**Comment by pkubatrh** [comment]
@phracek PTAL

[test][test-openshift]

**Comment by pkubatrh** [comment]
[test][test-openshift]

**Comment by pkubatrh** [comment]
[test][test-openshift]

**Comment by pkubatrh** [comment]
[test-openshift]

**Comment by pkubatrh** [comment]
[test][test-openshift]

**Comment by pkubatrh** [comment]
[test][test-openshift]

**Comment by pkubatrh** [comment]
[test-all]

**Comment by pkubatrh** [comment]
[test][test-openshift]

File: specs/multispec.yml [hunk 1]
```diff
@@ -120,6 +120,8 @@ matrix:
     - distros:
         - fedora-31-x86_64
         - fedora-34-x86_64
+        - rhel-7-x86_64
```
Comment (phracek) [inline specs/multispec.yml:123]: This is for version 10 and not 9.6 as is written in PR title.

But only nitpick from my side.

**Review by phracek — CHANGES_REQUESTED** [review]

File: specs/multispec.yml (hunk 1 above)
Comment (pkubatrh) [reply specs/multispec.yml]: Ah right, rebasing the PR must have thrown it into a different version. Should be fixed now

**Comment by phracek** [comment]
[test][test-openshift-3]

File: specs/multispec.yml (hunk 1 above)
Comment (phracek) [reply specs/multispec.yml]: +1

**Comment by phracek** [comment]
[test-openshift]

**Comment by phracek** [comment]
[test-openshift-4]

**Comment by phracek** [comment]
OpenShift 4 cluster timeout it. Let's try it again

[test-openshift-4]

**Review by phracek — APPROVED** [review]
LGTM

**Pull request merged**